ga4gh/_protocol_definitions.py

```diff
@@ -746,6 +863,84 @@ def __init__(self, **kwargs):
         """
 
 
+class FeatureGroup(ProtocolElement):
```

Comment:
This is part of RNA and represents an associated group of quantifications - for example a collection of transcripts from the same gene.  If the name is too confusing due to overload of the word 'feature' we can change it in future.